### PR TITLE
[14.0] purchase_quick_intercompany: add

### PR DESCRIPTION
**Pull request by kevinkhao**
This module does two things:
* In the purchase_quick interface, adds stock levels for the selling company
* Adds a configurable default UoM for intercompany purchase
- [x] depend on https://github.com/OCA/multi-company/pull/315

### CODE REVIEW

**Comment by sebastienbeau** [comment]
@kevinkhao can you update your PR as dependency have been merged ? Thanks

**Comment by kevinkhao** [comment]
> @kevinkhao can you update your PR as dependency have been merged ? Thanks

Rebased and removed test-requirements.txt

File: purchase_quick_intercompany/models/product_product.py [hunk 1]
```diff
@@ -23,7 +23,7 @@ def _compute_quick_stock_level(self):
                 purchase = self.env["purchase.order"].browse(
                     self.env.context.get("parent_id")
                 )
-                seller_partner = purchase.partner_id
+                seller_partner = purchase.partner_id.commercial_partner_id
```
Comment (Kev-Roche) [inline purchase_quick_intercompany/models/product_product.py:26]: @kevinkhao, @sebastienbeau
A suggestion in order to adapt seller_partner if he is a contact.

**Review by sebastienbeau — APPROVED** [review]
LGTM code review and test

**Review by Kev-Roche — APPROVED** [review]

**Pull request merged**